examples/NativeMessaging/App.js

```diff
@@ -18,6 +18,7 @@ import {
 } from 'stream-chat-react-native';
 
 import { useStreamChatTheme } from './useStreamChatTheme';
+import { GestureHandlerRootView } from 'react-native-gesture-handler';
```

Comment:
It would be better if you remove changes from NativeMessaging app, since we are dropping this app in https://github.com/GetStream/stream-chat-react-native/pull/1199